.goreleaser.yml

```diff
@@ -12,14 +16,13 @@ builds:
       - darwin
     goarch:
       - amd64
-
-archives:
-  - replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+    hooks:
+      pre:
+        - cmd: bash -c "GOOS={{ .Os }} GOARCH={{ .Arch }} packr2"
```

Comment:
The last time I tried using packr, it only worked if run in the folder where packr is used (https://github.com/hermeznetwork/hermez-node/blob/d5d2812cab0dd8c8eec3e65c8e3e0dc09f96cb29/db/utils.go#L34)
I believe this will run packr in the root of the project.  Have you checked moving this binary to another system and running it?

Reply:
the folder was specified in the next line:

https://github.com/hermeznetwork/hermez-node/pull/697/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689R22

Reply:
Ah! I missed that, thanks!